src/lesson6/Traingle.java

```diff
@@ -0,0 +1,25 @@
+package lesson6;
+
+public class Traingle implements Figures {
+    private double a;
+    private double b;
+    private double c;
+
+    public Traingle(double a, double b, double c) {
+        this.a = a;
+        this.b = b;
+        this.c = c;
+
+    }
+
+    @Override
+    public double calculaterPerimetr() {
+        return a + b + c;
+    }
+
+    @Override
+    public double calculaterArea() {
+        return Math.sqrt(calculaterPerimetr() / 2 * (calculaterPerimetr() / 2 - a) * (calculaterPerimetr() / 2 - b) *
```

Comment:
it's working fine, but from performance side it's nok as you are calling calculatePerimitr 4 times, instead I propose you such implementation:
@Override
    public double calculaterArea() {
        double perimeter = calculaterPerimetr();
        return Math.sqrt(perimeter / 2 * (perimeter / 2 - a) * (perimeter / 2 - b) *
                (perimeter / 2 - c));
    }
}
In such way the same calculation is performed, but code is much faster. This is general approach in such situation, if some method with returning value is called 2 and more times some local variable should be introduced